tasks/windows.ps1

```diff
@@ -0,0 +1,82 @@
+[CmdletBinding()]
+param(
+  # NOTE: init.json cannot yet be shared, so must have windows.json / windows.ps1
+  [Parameter(Mandatory = $true)]
+  [String]
+  $Name,
+
+  [Parameter(Mandatory = $true)]
+  [ValidateSet('start', 'stop', 'restart', 'status')]
+  [String]
+  $Action
+)
+
+$ErrorActionPreference = 'Stop'
+
+function Invoke-ServiceAction($Service, $Action)
+{
+  $inSyncStatus = 'in_sync'
```

Comment:
Rather than returning `in_sync`, I think it would be more useful to return the initial service state and the end service state after performing an action. I followed the model of the existing Puppet based task, but returning `restarted` is a bit nebulous, for instance - given a service start could fail, but this may not  be accurately represented in the return `status` (note that a failure to start should be an `Exception` which is currently handled and returned in an `_error` object, but there may be a situation where a start results in `StartPending`).

/cc @ccaum / @kenazk / @tylerpace

Reply:
+1 I'll raise this issue in the main Puppet based one too

Reply:
@ccaum should we get a ticket filed for this / a PR spiked?

Reply:
I filed https://tickets.puppetlabs.com/browse/MODULES-6931 to track the normalization. If we decide *not* to do that, we can close it with a note.

Reply:
I tossed up a quick spike at https://github.com/puppetlabs/puppetlabs-service/pull/29